.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,154 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  
+  cmake_test:
```

Comment:
Think it'd be good to add `name` to all the jobs. There are many similarly named / command jobs so a descriptive comment will help understand them both in this config and in the results.

Reply:
I added more descriptive names to each job. Is that what you were looking for?

Reply:
Yup these are very helpful, thanks!